src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextFieldUI.java

```diff
@@ -86,7 +86,7 @@ protected String getPropertyPrefix() {
      * Creates a view (FieldView) based on an element.
```

Comment:
```suggestion
     * Creates a view based on an element.
```
For consistency, if we don't document the types.

Reply:
ok